### ISSUE TITLE
Compatibily with instantsearch.js v1.0.0 ?

### ISSUE THREAD

**Issue by Spone** [instantsearch/instantsearch-googlemaps#1]
I have these packages in my `package.json`:

```
"instantsearch.js": "^1.0.0",
"instantsearch-googlemaps": "^1.1.2",
```

When I `npm install`, I have this error:

```
npm ERR! peerinvalid The package instantsearch.js does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer instantsearch-googlemaps@1.1.2 wants instantsearch.js@^0.10.0
```

Shouldn't instantsearch-googlemaps be compatible with the instantsearch.js v1.0.0 ?

Thanks

**Comment by vvo** [comment]
oops! this slipped of my todo. Doing it right now. Thanks

**Comment by vvo** [comment]
Ok released as 1.1.3:

``` sh
npm install instantsearch-googlemaps@1.1.3 --save-dev
```

**Comment by Spone** [comment]
:+1:
